website/docs/releases.mdx

```diff
@@ -9,6 +9,37 @@ import TierLabel from "./_components/TierLabel";
 :::note
 This page details the changes for Weave GitOps Enterprise and its associated components. For Weave GitOps OSS - please see the release notes on [GitHub](https://github.com/weaveworks/weave-gitops/releases).
 :::
+## v0.16.0
+2023-02-02
+### Highlights
+
+#### Create External Secrets via WGE UI
+- It's becoming easier to create new external secret CR through the UI instead of writing the whole CR yaml.
+- The creation form will help users choose which cluster to deploy the External Secret to and which secret store to sync the secrets from. 
+- It's all done in GitOps way. 
+
+#### Show PR Links from Pipline details
+- Users will be able to follow a link in the Pipeline Details Page to view open pull requests from given environment within their pipeline.
+
+#### Plan Button in Terraform
+- Adding add plan button in the terraform plan page to enable users to re-plan changes made. 
```

Comment:
```suggestion
- Adding **Add Plan** button in the terraform plan page to enable users to re-plan changes made. 
```

Reply:
Doesnt have to necessarily be bold, just anything that would make the button name clear in the sentence